.storybook/main.js

```diff
@@ -33,31 +33,16 @@ module.exports = {
   ],
   features: {
     postcss: false,
-    emotionAlias: false,
   },
   core: {
     builder: 'webpack5',
+    disableTelemetry: true,
   },
   webpackFinal: createWebpackConfig,
   managerWebpack: createWebpackConfig,
 };
 
 function createWebpackConfig(config) {
-  // Transpile all node_modules under the @sumup/* namespace.
-  config.module.rules = config.module.rules.map((rule) => {
-    // Modify all rules that apply to story files.
-    if (
-      rule.test &&
-      rule.test instanceof RegExp &&
-      ['.js', '.ts', '.tsx'].some((extension) => rule.test.test(extension))
-    ) {
-      return {
-        ...rule,
-        exclude: /node_modules\/(?!(@sumup|acorn-jsx)\/).*/,
-      };
-    }
-    return rule;
-  });
```

Comment:
I don't think that we need this anymore since we dropped IE support. I'll test the preview build on the browsers we still support